lib/doorkeeper-jwt.rb

```diff
@@ -16,9 +16,11 @@ def self.generate(opts = {})
 
     def self.token_payload(opts = {})
       Doorkeeper::JWT.configuration.token_payload.call opts
+
     end
 
-    def self.secret_key
+    def self.secret_key(opts)
```

Comment:
`private` (on line 15) does not make singleton methods private. Use `private_class_method` or `private` inside a `class << self` block instead.